tegola_st_asmvt.toml

```diff
@@ -421,7 +421,7 @@ center = [-76.275329586789, 39.153492567373, 8.0] # optional center value. part
 	name = "populated_places"
 	provider_layer = "osm.ne_10m_populated_places"
 	min_zoom = 5
-	max_zoom = 16
+	max_zoom = 20
```

Comment:
max_zoom should be `16` as the UI will overzoom on zoom 16. You mention that another tegola example config is using 20? LMK where you saw that and let's update it to use max_zoom of 16. This is a very important detail in my mind as the amount of tiles you need to create is massive above 16 and it's not necessary.

Reply:
Right here: https://github.com/go-spatial/tegola-osm/blob/master/tegola.toml#L370

Reply:
Got it. Ok so that's a bug. We should align everything to zoom 16. If you want to do this with this PR I can get this PR approved. Thanks for the help, and again apologies for the slow response.

Reply:
My pleasure. Hope this helps!

Reply:
@diehl looks like the `max_zoom` is still set to 20 instead of 16 in [tegola_st_asmvt.toml](https://github.com/go-spatial/tegola-osm/pull/68/files#diff-4ebc41b1db1c7b3aaa86fd5b3651d2a2b21c41877eb1d34bec12eaed26f30470). Can you please adjust?

Reply:
@ARolek Strange. I'm not seeing that. https://github.com/diehl/tegola-osm/blob/mvt_toml_fix/tegola_st_asmvt.toml#L424 shows `max_zoom` at 16.

Reply:
Check further down the file, for example: https://github.com/diehl/tegola-osm/blob/mvt_toml_fix/tegola_st_asmvt.toml#L540

Reply:
@ARolek Yes those are different entries. Do you want all of the `max_zoom` entries to be no greater than 16?

Reply:
Yep. We should never have a max zoom over 16.

Reply:
@ARolek Got it. I think we're good to go now. Let me know if I missed anything!